docs/en/advanced_guides/conventions.md

```diff
@@ -2,6 +2,38 @@
 
 Please check the following conventions if you would like to modify MMDetection as your own project.
 
+## Image pipeline input shape
+
+In OpenMMLab 2.0, the image pipeline input shape is `(width, height)`, the shape of the pipeline output is `(height, width)` and the shape is also always `(height, width)` in model. The common fields are as follows:
+
+- img_shape: (height, width)
```

Comment:
Specifically, in the results processed by each data transform pipeline, the fields and their value meaning is as below: